### DISCUSSION TITLE
Invalid signature on JWT.io

### DISCUSSION THREAD

**Discussion by jiveyTO** [nextauthjs/next-auth#1531] [Help] [answered]
**Your question**
I can validate my req and secret with the following, jwt being imported from next-auth/jwt
jwt.getToken({ req, secret })

However when I take req.cookies['next-auth.session-token']  and the same secret and enter them into JWT.io, I get an invalid signature.

**What are you trying to do**
I'm trying to validate on JWT.io because I'm using jsonwebtoken on my API server and won't have access to next-auth/jwt

**Reproduction**

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [x] Found the documentation helpful
* [x] Found documentation but was incomplete
* [ ] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by StefanSelfTaught** [comment, marked as answer]
This might help: https://github.com/nextauthjs/next-auth/discussions/1039#discussion-1336033
